tools/ramses-logic-viewer/Arguments.h

```diff
@@ -65,13 +66,19 @@ struct Arguments
         }
 
         const std::string argExec = "--exec=";
+        const std::string argMultiSample = "--multi-sample=";
 
         for (const std::string argString : args)
         {
             if (argString == "--no-offscreen")
             {
                 noOffscreen = true;
             }
+            else if (argString.find(argMultiSample, 0) == 0)
+            {
+                std::string sampleValue = argString.substr(argString.find("=") + 1);
```

Comment:
```suggestion
                std::string sampleValue = argString.substr(argString.find('=') + 1);
```

Otherwise looks good (this causes a clang-tidy warning)

Reply:
fixed